### DISCUSSION TITLE
Dexcom Share: Ability to Specify Seconds Until Polling Retry

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3347] [General]
@olorinmaia I am puzzled.
https://github.com/NightscoutFoundation/xDrip/issues/1849#issuecomment-1947152994
Does this mean when you look at what is missing in xDrip, you believe that issue is the worst?

Do you know that - 

1- the persistent high alert does not respect the chosen volume profile?  So, if you are persistently high and rely on the persistent high alert, but accidentally lowered your phone volume, you may never hear the alert all night.

2- sometimes the start, or stop, flag is placed at an extremely high value causing the y axis to be extended dramatically?

3- the persistent high alert (similar to the forecast low) does not use a dedicated threshold?  It uses the statistic high threshold.  So, every time you want to create a report, you will have to change the setting and remember to change it back.

4- Libre users have difficulty calibrating because of the safety measures in the xDrip code that effectively make calibrations confusing?

5- xDrip does not offer audio focus for "Speak Readings"?

6- how many issues we have that people swear they experience, but we cannot recreate?

7- xDrip does not offer custom vibration patterns for the alerts?  So, someone who is deaf has no way of knowing which alert is triggered until they get their phone out so that they can look at it?

8- xDrip is incapable of importing a database?  It can only overwrite it.

9- no one is supporting WearOS feature requests in xDrip?

10- the smart alert and smart snooze are effectively broken?

11- we have a database corruption issue we are not sure the cause of?  https://github.com/NightscoutFoundation/xDrip/issues/1280

The problem with fixing many of these issues is like fixing a cruise ship full of passengers afloat in the middle of the ocean!

xDrip already offers xDrip sync as well as Nightscout.  
I cannot tell you what to do because you are a volunteer.  If I could, I would encourage you to focus on something else, easy to find, with a significantly higher priority.
I have too much on my plate.  There are things I know I have to do, but don't find time.

**Comment by olorinmaia** [comment]
No, not at all. I understand very well that there are several other more important issues. And that your focus will tend to them first. 

In my case im still in the beginner stage of coding, so what type of issues i can assist with is limited. And my time is also limited due to work and family. 😅 I think you misunderstood me abit. I appreciate your work on xdrip! 👍

**Reply by Navid200** [reply]
This issue is related to AAPS: https://github.com/NightscoutFoundation/xDrip/issues/1331

Do you want to have a look to see if you may be interested to help?
